pyaedt/edb_core/siwave.py

```diff
@@ -266,8 +266,8 @@ def create_port_between_pin_and_layer(
 
         Returns
         -------
-        PadstackInstanceTerminal
-            return the created terminal.
+        terminal
+            Created terminal (PadsatckInstanceTerminal).
```

Comment:
```suggestion
        PadstackInstanceTerminal
            Created terminal.
```
This suggestion assumes that we want to follow what is normally done. The name of the object returned is in the first line with a description of this object in the second line. If this is ignored, note that was is there currently has a typo in (Pad**sa**tckInstanceTerminal).

Reply:
Yes the typo is actually because it is returning the EDB object from .NET.